test/e2e/delete_test.go

```diff
@@ -4,47 +4,45 @@ import (
 	"context"
 	"testing"
 
-	operator "github.com/openshift/windows-machine-config-operator/pkg/apis/wmc/v1alpha1"
+	mapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
 	"github.com/stretchr/testify/require"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 func deletionTestSuite(t *testing.T) {
 	t.Run("Deletion", func(t *testing.T) { testWindowsNodeDeletion(t) })
-	t.Run("Status", func(t *testing.T) { testStatusWhenSuccessful(t) })
-	t.Run("ConfigMap validation", func(t *testing.T) { testConfigMapValidation(t) })
-	t.Run("Secrets validation", func(t *testing.T) { testValidateSecrets(t) })
 }
 
 // testWindowsNodeDeletion tests the Windows node deletion from the cluster.
 func testWindowsNodeDeletion(t *testing.T) {
 	testCtx, err := NewTestContext(t)
 	require.NoError(t, err)
 
-	// get WMCO custom resource
-	wmco := &operator.WindowsMachineConfig{}
-	// Get the WMCO resource called instance
-	err = framework.Global.Client.Get(context.TODO(), types.NamespacedName{Name: wmcCRName,
-		Namespace: testCtx.namespace}, wmco)
-	if err != nil && k8serrors.IsNotFound(err) {
-		// We did not find WMCO CR, let's recreate it. This is a possibility when the creation and deletion tests are
-		// run independently.
-		wmco, err = testCtx.createWMC(gc.numberOfNodes, gc.sshKeyPair)
-		require.NoError(t, err)
-	}
+	// The second item in the slice is the Windows MachineSet we're interested in scaling down.
+	require.Len(t, gc.machineSetNames, 2)
```

Comment:
Why `2` hard coded here? Isn't `2` an input to the script?

Reply:
Nope, not in this case. `2` becomes the number of replicas for the Windows MachineSet with appropriate label. We are creating 2 different machinesets to cover -ve scenario.

Reply:
Will this test fail if the input is greater than 2?

Reply:
Nope  but we changed the logic a bit.